src/docs/command-line.dox

```diff
@@ -202,6 +202,12 @@ the \c -f option re-directs the output to the specified file.
 Dump in JSON (<a href="http://www.json.org">JavaScript Object Notation</a>)
 format.
 
+@par \c -k
+The specific key to look for.
+
+@par \c -n
+Use \c search_near instead of \c search when a key is given through the \c -k option.
+
```

Comment:
Do we consider using a single command argument (e.g. upper case "K") to control key + search_near? I believe search_near won't be valid without specifying a key on the command line. It can also save a command argument typing and help in the long run if this new command argument becomes popular enough :)

Reply:
@marcbutler, what do you think about this suggestion?

Reply:
It would definitely be "clean" however without the second piece of information it is impossible to distinguish whether the key is in a raw string (bytes) format, or a hex format.

Reply:
Ok, let's leave it for now.